libs/island-ui/core/src/lib/PdfViewer/PdfViewer.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useState } from 'react'
+import { Box } from '../Box/Box'
+import { pdfjs, Document, Page } from 'react-pdf'
```

Comment:
I don't know for sure but according to [this](https://github.com/wojtekmaj/react-pdf#webpack) I think you can skip `pdfjs` for the worker stuff by importing this way:

```javascript
import { Document, Page } from 'react-pdf/dist/esm/entry.webpack';
```

Reply:
Yes we actually imported it from 'react-pdf/dist/esm/entry.webpack' first but got some errors on the automatic checks so we changed it in consultation with Eiríkur :)